lang/sql/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Style Guide for SQL
+pageid: lang-sql
```

Comment:
My version of BSD version of sed doesn't play nice with that script, and I'm not having much luck troubleshooting D: